stages/org.osbuild.grub2.iso

```diff
@@ -53,6 +53,12 @@ SCHEMA_2 = """
     },
     "vendor": {
         "type": "string"
+    },
+    "timeout": {
+      "description": "Grub timeout in seconds",
+      "type": "integer",
+      "minimum": 0,
+      "default": 0
```

Comment:
In the `grub2` stage we put these in a `config` section. Might be a good idea to do the same to not have too many different variants? https://github.com/osbuild/osbuild/blob/9d1eb6cecc0673b8952d1b1f512e44fb8de038f0/stages/org.osbuild.grub2#L191-L217

Reply:
fixed